src/test/java/io/vertx/junit5/VertxExtensionTest.java

```diff
@@ -17,7 +17,8 @@
 package io.vertx.junit5;
 
 import io.vertx.core.Vertx;
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
I don't think that those imports are needed, but this may be a side effect of #26